### ISSUE TITLE
How to acquire the file of ' vqa_img_feature_train.pickle'?

### ISSUE THREAD

**Issue by coral3333** [AndersonStra/MuKEA#1]
How to acquire the file of ' vqa_img_feature_train.pickle'?

**Comment by AndersonStra** [comment]
Image features are recommended to download  from the LXMERT's  [repo](https://github.com/airsplay/lxmert#google-drive) or bottom-up attention 's [repo](https://github.com/peteanderson80/bottom-up-attention#pretrained-features),  We reorganized the storage structure of features as 
```
vqa_img_feature_train.pickle{
"image_id":{'feats': features, 'sp_feats': spatial features}
}
```
The pre-trained LXMERT model expects these spatial features to be normalized bounding boxes on a scale of 0 to 1

The normalized process can refer to the [prepare_img.py](https://github.com/AndersonStra/MuKEA/blob/main/vqa_v2_pretrain/prepare_img.py)

we will provide the shell to help generate the pickle file from the original bottom-up attention file further.

**Comment by AndersonStra** [comment]
I have updated the script to help generate the pickle file from the original bottom-up attention file.

**Comment by coral3333** [comment]
谢谢你的回复，为你的工作点赞！！祝你一切顺利！！




dushanshan
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "AndersonStra/MuKEA"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年4月6日(星期三) 下午3:59
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [AndersonStra/MuKEA] How to acquire the file of ' vqa_img_feature_train.pickle'? (Issue #1)





 
I have updated the script to help generate the pickle file from the original bottom-up attention file.
